requests_lb/requests_lb.py

```diff
@@ -135,34 +136,34 @@ def _retry_request(self, fn, target, **kw):
         Send the given request, as defined by the `fn` argument to the given
         target.
         """
+        _max_request_retries = self._max_request_retries
```

Comment:
Single underscore prefix is convention for 'internal use', and since this is a local variable it should not be necessary.